client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -268,6 +372,7 @@ export const SitesEllipsisMenu = ( {
 				<SiteMenuGroup>
 					{ site.launch_status === 'unlaunched' && <LaunchItem { ...props } /> }
 					<SettingsItem { ...props } />
+					{ isEnabled( 'dev/developer-ux' ) && <DeveloperSettingsSubmenu { ...props } /> }
 					<ManagePluginsItem { ...props } />
 					{ showHosting && <HostingConfigItem { ...props } /> }
```

Comment:
Should we remove Hosting Configuration link if we shod more granular links in the new submenu?

Reply:
Good question. I had the same thought.
My reasoning for not doing it is that we would want to keep the `Hosting Configuration` link for Free sites.
From my point of view, it makes sense to remove it for atomic sites. I'll move the discussion to P2. 👌 
Thank you!